spec/acceptance/bundle_management_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper_acceptance'
+require 'fileutils'
+
+describe 'Managing Gemfile dependencies' do
+  before(:all) do
+    # Do everything in a clean tmpdir.
+    @old_pwd = Dir.pwd
+    @tmpdir = Dir.mktmpdir('pdk-acceptance')
+
+    Dir.chdir(@tmpdir)
```

Comment:
This will only work locally, but not when using beaker. We'll need to extend beaker-testmode_switcher's API to provide better support for such operations.